js/demo-app/src/index.js

```diff
@@ -4,22 +4,54 @@ import coordinator from "iframe-coordinator/host.js";
 import toastada from "toastada";
 
 const TOAST_LEVELS = ["info", "success", "error"];
+const TOP_ROUTE_EXTRACTOR = /^#?(\/[^\/]+).*/;
+const NAV_CONFIGS = [
+  {
+    id: 'client1',
+    title: 'Component 1',
+    url: new URL("/components/example1/", window.location).toString(),
+    assignedRoute: "/one",
+    children: ['first', 'second']
+  },
+  {
+    id: 'client2',
+    title: 'Component 2',
+    url: new URL("/components/example2/", window.location).toString(),
+    assignedRoute: "/two",
+    children: ['first', 'second']
+  },
+  {
+    id: 'wikipedia',
```

Comment:
Even though it doesn't participate in routing, I thought this helps to show users we really are dealing with iframes and embeddable content.  It also can help make sure we're handling fq paths in the routing correctly.

If there's any opposition to this, easy to pull out.  Not tied to it.